v22.1/take-and-restore-locality-aware-backups.md

```diff
@@ -117,29 +117,43 @@ To restore from a specific backup, use [`RESTORE FROM {subdirectory} IN ...`](re
 
 ## Create an incremental locality-aware backup
 
-If you backup to a destination already containing a [full backup](take-full-and-incremental-backups.html#full-backups), an [incremental backup](take-full-and-incremental-backups.html#incremental-backups) will be appended to the full backup in a subdirectory. For example:
+If you backup to a destination already containing a [full backup](take-full-and-incremental-backups.html#full-backups), an [incremental backup](take-full-and-incremental-backups.html#incremental-backups) will be appended to the full backup in a subdirectory. When you're taking an incremental backup, you must ensure that the incremental backup localities match the full backup localities otherwise you will receive an error. Alternatively, take another full backup with the matching localities before running the incremental backup. 
 
-{% include_cached copy-clipboard.html %}
-~~~ sql
-> BACKUP INTO LATEST IN
-	  ('s3://us-east-bucket?COCKROACH_LOCALITY=default', 's3://us-west-bucket?COCKROACH_LOCALITY=region%3Dus-west');
-~~~
+There is different syntax for taking an incremental backup depending on where you need to store the backups:
 
-{{site.data.alerts.callout_info}}
-When [restoring from an incremental locality-aware backup](#restore-from-an-incremental-locality-aware-backup), you need to include **every** locality ever used, even if it was only used once.
+- To append your incremental backup to the full backup in the [`incrementals` directory](take-full-and-incremental-backups.html#backup-collections):
 
-It is recommended that the same localities be included for every incremental backup in the series of backups; however, only the `default` locality is required.
-{{site.data.alerts.end}}
+	{% include_cached copy-clipboard.html %}
+	~~~ sql
+	BACKUP INTO LATEST IN
+		('s3://us-east-bucket?COCKROACH_LOCALITY=default', 's3://us-west-bucket?COCKROACH_LOCALITY=region%3Dus-west');
+	~~~
 
-And if you want to explicitly control where your incremental backups go, specify the subdirectory in the `BACKUP` statement:
+	{{site.data.alerts.callout_info}}
+	When [restoring from an incremental locality-aware backup](#restore-from-an-incremental-locality-aware-backup), you need to include **every** locality ever used, even if it was only used once.
 
-{% include_cached copy-clipboard.html %}
-~~~ sql
-> BACKUP INTO {subdirectory} IN
-		('s3://us-east-bucket?COCKROACH_LOCALITY=default', 's3://us-west-bucket?COCKROACH_LOCALITY=region%3Dus-west');
-~~~
+	It is necessary that the same localities be included for every incremental backup in the series of backups; however, only the `default` locality is required.
+	{{site.data.alerts.end}}
+
+- If you want to explicitly control the subdirectory for your incremental backups:
```

Comment:
nit: Ideally all three bullet points introduced as "there is different syntax depending:" are introduced in the same way. I'm a big fan of the direct approach you used in the first one:

> - To append your incremental backup to the full backup in the [`incrementals` directory]:
> - To explicitly control the subdirectory for your incremental backup:
> - To append your incremental backup to the full backup using the [`incremental_location`] option to send your ...

Reply:
Done.